test/IECore/SplineTest.py

```diff
@@ -255,6 +255,13 @@ def computeIntegrals( s, interval = None ):
 		( integral, summedArea ) = computeIntegrals(s, [12,20])
 		self.assertAlmostEqual( integral, summedArea, 3 )
 
+		# testing linear basis integral with 4 control points
+		s = IECore.Splineff( IECore.CubicBasisf.linear(), ( ( 0, 1 ), ( 10, 2 ), ( 20, 0 ), ( 21, 2 ) ) )
+		( integral, summedArea ) = computeIntegrals(s)
```

Comment:
Not sure why Travis didn't run, but I ran the tests locally and got this crash from the `computeIntegrals()` call :

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libIECorePython.dylib         	0x000000010459f810 IECore::Spline<float, float>::solve(float, float*) const + 272
1   libIECorePython.dylib         	0x00000001048f1493 IECore::Spline<float, float>::operator()(float) const + 35
2   libIECorePython.dylib         	0x00000001048f5e6d boost::python::detail::caller_arity<2u>::impl<float (IECore::Spline<float, float>::*)(float) const, boost::python::default_call_policies, boost::mpl::vector3<float, IECore::Spline<float, float>&, float> >::operator()(_object*, _object*) + 173
3   libboost_python.dylib         	0x00000001016703a6 boost::python::objects::function::call(_object*, _object*) const + 822
4   libboost_python.dylib         	0x0000000101672f0a boost::detail::function::void_function_ref_invoker0<boost::python::objects::(anonymous namespace)::bind_return, void>::invoke(boost::detail::function::function_buffer&) + 26
5   libboost_python.dylib         	0x0000000101679013 boost::python::detail::exception_handler::operator()(boost::function0<void> const&) const + 67
6   libIECorePython.dylib         	0x0000000104201539 bool boost::_bi::list3<boost::arg<1>, boost::arg<2>, boost::_bi::value<void (*)(IECore::Exception const&)> >::operator()<bool, boost::python::detail::translate_exception<IECore::Exception, void (*)(IECore::Exception const&)>, boost::_bi::list2<boost::python::detail::exception_handler const&, boost::function0<void> const&> >(boost::_bi::type<bool>, boost::python::detail::translate_exception<IECore::Exception, void (*)(IECore::Exception const&)>&, boost::_bi::list2<boost::python::detail::exception_handler const&, boost::function0<void> const&>&, long) + 25
7   libIECorePython.dylib         	0x0000000104201515 boost::detail::function::function_obj_invoker2<boost::_bi::bind_t<bool, boost::python::detail::translate_exception<IECore::Exception, void (*)(IECore::Exception const&)>, boost::_bi::list3<boost::arg<1>, boost::arg<2>, boost::_bi::value<void (*)(IECore::Exception const&)> > >, bool, boost::python::detail::exception_handler const&, boost::function0<void> const&>::invoke(boost::detail::function::function_buffer&, boost::python::detail::exception_handler const&, boost::function0<void> const&) + 37
8   libboost_python.dylib         	0x0000000101678dc3 boost::python::handle_exception_impl(boost::function0<void>) + 51
9   libboost_python.dylib         	0x0000000101672a53 boost::python::objects::function_call(_object*, _object*, _object*) + 83
```

Is it just me?